portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/FormEvents.testcase

```diff
@@ -399,8 +399,8 @@ definition {
 				locator1 = "FormFields#TEXT_FIELD");
 		}
 
-		task ("Stay 2 seconds") {
-			Pause(locator1 = "2000");
+		task ("Stay 5 seconds") {
```

Comment:
Hey @calvinctvo, I think we could put this task here to keep the pattern. What do you think?

**task ("Wait for sending requests")**